pkgs/development/python-modules/zxcvbn-rs-py/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  rustPlatform,
+  cargo,
+  rustc,
+}:
+
+buildPythonPackage rec {
+  pname = "zxcvbn-rs-py";
+  version = "0.1.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "fief-dev";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-hy23vyS78B78B5NHk0CU11awfpJD2aeD9dEWbCR9q6c=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock { lockFile = ./Cargo.lock; };
+
+  build-system = [ rustPlatform.maturinBuildHook ];
```

Comment:
Nit: I think this should go into nativeBuildInputs, as you're placing the hook here rather than the package...but Maturin still is the build system so 🤷